### PR TITLE
Show default value when printing options

### PR DESCRIPTION
**Pull request by fampinheiro**
This PR adds the default option in grey after the argument description.

```
Options:

  -c, --currency       currency (eur)
```

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -213,6 +213,10 @@ exports.usage = function (definition, usage, options) {
         }
 
         var formattedDesc = def.description ? color.gray(def.description) : '';
+        if (def.default) {
+            formattedDesc += formattedDesc.length ? ' ' : '';
+            formattedDesc += color.gray('(' + def.default + ')');
```
Comment (sericaia) [inline lib/index.js]: I just don't know if having it in grey is the best option, but I will accept it

**Comment by danielb2** [comment]
Any chance you can do a new release so we can see the benefits of this PR?

After all, it's been five months

**Comment by sericaia** [comment]
@danielb2 Done ;)

**Comment by danielb2** [comment]
Thank you :) :+1:

**Pull request merged**